tools/install_appveyor.py

```diff
@@ -53,65 +50,74 @@ def run_command(raw_command, directory=None, verbose=True):
+        run_command(r'7z x -aoa -oC:\\ python.7z', verbose=False)
+        run_command(r'mv C:\\Python27 C:\\Python27-x64', verbose=False)
     else:
         raise RuntimeError('Unsupported Python build: ' + BUILD_TYPE)
-    python_package = r'python' + python_version + r'_mingw_64.7z'
-    boost_python_package = r'boost_python_' + python_version + r'_mingw_64.7z'
-    # Remove any existing Python installation.
-    rm_fr(r'c:\\Python' + python_version)
+
     # Set paths.
-    pinterp = r'c:\\Python' + python_version + r'\\python.exe'
-    pip = r'c:\\Python' + python_version + r'\\scripts\\pip'
-    twine = r'c:\\Python' + python_version + r'\\scripts\\twine'
-    pykep_install_path = r'C:\\Python' + \
-        python_version + r'\\Lib\\site-packages\\pykep'
-    # Get Python.
-    wget(r'https://github.com/bluescarni/binary_deps/raw/master/' +
-         python_package, 'python.7z')
-    run_command(r'7z x -aoa -oC:\\ python.7z', verbose=False)
-    # Get Boost Python.
-    wget(r'https://github.com/bluescarni/binary_deps/raw/master/' +
-         boost_python_package, 'boost_python.7z')
-    run_command(r'7z x -aoa -oC:\\ boost_python.7z', verbose=False)
+    pinterp = r"C:\\" + python_folder + r'\\python.exe'
+    pip = r"C:\\" + python_folder + r'\\scripts\\pip'
+    twine = r"C:\\" + python_folder + r'\\scripts\\twine'
+    module_install_path = r"C:\\" + python_folder + r'\\Lib\\site-packages\\pykep'
     # Install pip and deps.
+    run_command(pinterp + r' --version', verbose=True)
     wget(r'https://bootstrap.pypa.io/get-pip.py', 'get-pip.py')
     run_command(pinterp + ' get-pip.py --force-reinstall')
     if is_release_build:
         run_command(pip + ' install twine')
 
-# Set the path so that the precompiled libs can be found.
-os.environ['PATH'] = os.environ['PATH'] + r';c:\\local\\lib'
-
 # Proceed to the build.
 os.makedirs('build')
 os.chdir('build')
-common_cmake_opts = r'-DCMAKE_PREFIX_PATH=c:\\local -DCMAKE_INSTALL_PREFIX=c:\\local -DBUILD_SPICE=yes'
+common_cmake_opts = r'-DCMAKE_PREFIX_PATH=c:\\local -DCMAKE_INSTALL_PREFIX=c:\\local -DBUILD_SPICE=no '
 
 # Configuration step.
 if is_python_build:
-    run_command(r'cmake -G "MinGW Makefiles" ..  -DBUILD_PYKEP=yes -DCMAKE_BUILD_TYPE=Release ' + common_cmake_opts + r' -DBoost_PYTHON' + (python_version[0] if python_version[0] == '3' else r'') + r'_LIBRARY_RELEASE=c:\\local\\lib\\libboost_python' +
-                (python_version[0] if python_version[0] == '3' else r'') + r'-mgw62-mt-1_63.dll  -DPYTHON_EXECUTABLE=C:\\Python' + python_version + r'\\python.exe -DPYTHON_LIBRARY=C:\\Python' + python_version + r'\\libs\\python' + python_version + r'.dll')
+    run_command(r'cmake -G "MinGW Makefiles" .. -DCMAKE_BUILD_TYPE=Release ' + \
+        common_cmake_opts + \
+        r'-DBUILD_PYKEP=yes ' + \
+        r'-DBUILD_MAIN=no ' + \
+        r'-DBUILD_TESTS=no ' + \
+        r'-DBoost_INCLUDE_DIR=c:\\local\\include ' + \
```

Comment:
Any particular reason here for setting all the includes/libs from boost manually, instead of using the ``CMAKE_PREFIX_PATH`` variable set to ``c:\local``?

Reply:
Do not ask why but if I remove the direct link to boost, leaving only the ```CMAKE_PREFIX_PATH``` set, the build fails as it does not find the libs .... https://ci.appveyor.com/project/darioizzo/pykep/builds/24488071/job/85p6aal8our4tmiq#L164

Reply:
Looks a bit weird, it's not finding the libraries (finds only Boost Python as it was explicitly specified on the command line) but it does not stop the CMake run. Maybe serialization et al. are not marked as ``REQUIRED`` when ``find_package(Boost)`` is invoked?

Reply:
I am not a CMake expert, but in here,
https://github.com/darioizzo/pykep/blob/master/cmake_modules/PykepFindBoost.cmake#L20
is
where I think I make sure they are marked as required

Reply:
There's also another ``find_package()`` call earlier here:

https://github.com/darioizzo/pykep/blob/master/cmake_modules/PykepFindBoost.cmake#L3

Maybe that gets it confused? No idea...

Reply:
Tried that too, but the other find package is needed to determine the boost version used to give a name to the boost python lib ... same trick is also in pagmo2...... Unfortunately I do not understand whats gong on ... will merge this as is and keep it in mind for the other packages (dcgp, pagmo2) ....